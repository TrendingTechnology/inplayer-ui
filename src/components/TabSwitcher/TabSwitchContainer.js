import styled from 'styled-components';
import colors from 'config/colors';

const TabSwitchContainer = styled.div`
  border: 1px solid ${colors.gray};
  border-radius: 3px;
  margin: 0 auto;
  display: flex;
  overflow: hidden;
  text-align: center;
`;

export default TabSwitchContainer;
