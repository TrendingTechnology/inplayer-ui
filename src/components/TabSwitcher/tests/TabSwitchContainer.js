import { generateSnapshots } from 'helpers';

import TabSwitchContainer from '../TabSwitchContainer';

generateSnapshots(TabSwitchContainer);
