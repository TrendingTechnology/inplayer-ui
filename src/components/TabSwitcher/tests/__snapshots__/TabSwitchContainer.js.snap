// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabSwitchContainer renders correctly 1`] = `
.c0 {
  border: 1px solid #d7dde5;
  border-radius: 3px;
  margin: 0 auto;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  overflow: hidden;
  text-align: center;
}

<div
  className="c0"
/>
`;
